### PR TITLE
✨(social-badges) add social network badges template snippets

### PR DESCRIPTION
**Pull request by sveetch**
## Purpose

We need a way to include common social network badges in some
templates.

## Proposal

This add new templates to define social network badges for course,
blogpost pages and footer. For social network icons, we include also
an icon font builded from icomoon.io

## Todo

- [x] Snippet for course badges;
- [x] Snippet for blogpost badges;
- [x] Snippet for footer badges;
- [x] Build and include icon font and css for social network icons;

### CODE REVIEW

File: src/richie/apps/core/templates/social-networks/course-badges.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% spaceless %}
+{% blocktrans with page_title=page_title page_url=page_url asvar twitter_text %}Je viens de m'inscrire pour '{{ page_title }}' via @funmooc: {{ page_url }}{% endblocktrans %}
```
Comment (sampaccoud) [inline src/richie/apps/core/templates/social-networks/course-badges.html]: The messages here need to be generic richie messages.

File: src/richie/apps/core/templates/social-networks/course-badges.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{% blocktrans asvar mailto_subject %}Suivez un cours en ligne avec FUN{% endblocktrans %}
+{% blocktrans with page_title=page_title page_url=page_url asvar mailto_body %}Je viens de m'inscrire pour '{{ page_title }}' via FUN: {{ page_url }}{% endblocktrans %}
+<div class="social-badges">
+    <a href="https://www.facebook.com/france.universite.numerique" target="_blank" class="social-badges__item social-badges__item--facebook" title="{% trans "Facebook page" %}">
+        <span class="icon-facebook">F</span>
```
Comment (sampaccoud) [inline src/richie/apps/core/templates/social-networks/course-badges.html]: It would be nice to add the icons in the same PR WDYT?

Comment (sveetch) [reply src/richie/apps/core/templates/social-networks/course-badges.html]: yep, just wanted to quickly push something to preview

File: src/richie/apps/core/templates/social-networks/course-badges.html (hunk 1 above)
Comment (sveetch) [reply src/richie/apps/core/templates/social-networks/course-badges.html]: How to do them generic ? From settings ? But so how to reach this settings from template ?

Also from course to blogpost, message in share badges are not the same.

File: src/richie/apps/core/templates/social-networks/course-badges.html (hunk 2 above)
Comment (sampaccoud) [reply src/richie/apps/core/templates/social-networks/course-badges.html]: no I juste meant to change the sentence to not mention fun-mooc.
Do exactly what you wrote but replace the text by:
```
I just enrolled to the course "{{ page_title }}" on Richie: {{ page_url }}
```
Replace:
https://www.facebook.com/france.universite.numerique  
by https://www.facebook.com/example  (I checked that it does not exist, so ok...)

File: src/richie/apps/core/templates/social-networks/course-badges.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% spaceless %}
+{% blocktrans with page_title=page_title page_url=page_url asvar twitter_text %}Je viens de m'inscrire pour '{{ page_title }}' via @funmooc: {{ page_url }}{% endblocktrans %}
+{% blocktrans asvar mailto_subject %}Suivez un cours en ligne avec FUN{% endblocktrans %}
```
Comment (sampaccoud) [inline src/richie/apps/core/templates/social-networks/course-badges.html]:
```suggestion
{% blocktrans asvar mailto_subject %}Follow a course online with Richie{% endblocktrans %}
```

**Comment by sveetch** [comment]
## Badges preview for course detail

course

## Badges preview for blogpost detail

blogpost

## Badges preview for footer

footer

**Comment by sampaccoud** [comment]
@sveetch superb :smiley:

**Pull request merged**